Portfolio_Page.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="Style_of_Portfolio.css">
+</head>
+<body>
+    <header>
+        <section class="textbox">
+        <h1>My Portfolio</h1>
+        <nav>
+            <ul>
+                <li><a href="Home_Page.html" class="btn">Home</a></li>
+                <li><a href="Bio_Page.html" class="btn">Bio</a></li>
+                <li><a href="Experience_Page.html" class="btn">Experience</a></li>
+                <li><a href="Portfolio_Page.html" class="btn">Portfolio</a></li>
+            </ul>
+        </nav>
+        </section>
+    </header>
+    <section class="flex-container">
+        <div class="flex-item">
+            <h2>Chess Project</h2>
+            <ul>
+                <li>Showed off my multi-language skill as I combined my knowledge of HTML, CSS, and JS to accomplish.</li>
+                <li>Used AI to integrate a level system and adaptive computer player.</li>
+                <li>Working to create a database to store user information and game history.</li>
+                <li>Working to adapt a player level system to the game, similar to world Elo.</li>
+                <image src="https://wallpapertag.com/wallpaper/full/c/e/8/174002-gorgerous-chess-wallpaper-2605x1953-hd-1080p.jpg" alt="Chess Photo">
+            </ul>
+        </div>
+            <div class="flex-item">
+                <h2>SQL Query Analyzer</h2>
+                <ul>
+                    <li>Placeholder for future project </li>
+                        <li>Future SQL Project</li>
+                    <image src="https://tse2.mm.bing.net/th?id=OIP.CRFLi69c3c7nIBUJqgDzbAHaDd&pid=Api&P=0&h=220" alt="SQL Image">
+                </ul>
+        </div>
+        <div class="flex-item">
```

Comment:
I appreciate the semantic elements being used in this file! see if there are any others that you could use to continue to increase accessibility - https://www.w3schools.com/html/html5_semantic_elements.asp